src/components/page/Header/index.tsx

```diff
@@ -100,9 +103,15 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
       <div>
         <Navbar light={true} expand="md">
           <nav className="navbar navbar-expand-md navbar-light header-logo-navbar">
-            <Link to="/" className="navbar-brand">
-              <img src={logo} alt={WEBSITE_NAME} />
-            </Link>
+            {REACT_APP_PRODUCTION_INST === TH_PRODUCTION_INST ? (
+              <Link to="/" className="navbar-brand">
+                <img src={thaiLogo} alt={WEBSITE_NAME} style={{ transform: 'scale(2)' }} />
+              </Link>
+            ) : (
+              <Link to="/" className="navbar-brand">
+                <img src={logo} alt={WEBSITE_NAME} />
+              </Link>
+            )}
```

Comment:
How do you compare doing it this way Vs having the value of the `img` `src` come from a config?

Reply:
yeah I think using the src is more flexible.